main.go

```diff
@@ -127,26 +127,22 @@ func updatetemplateInformation(dcrdClient *dcrrpcclient.Client) {
 	templateInformation.BlockVersions = blockVersionsFound
 
 	// Calculate current block version and most popular version (and that percentage)
-	templateInformation.BlockVersionCurrent = int32(maxVersion)
+	templateInformation.BlockVersionCurrent = int32(3)
 	mostPopularBlockVersionCount := 0
 	// Range across rolling window block version results.  If any of the rolling window look back
 	// counts are greater than the required threshold then that is assured to be the current block
 	// version at point in the chain.
+
 	for i, blockVersion := range templateInformation.BlockVersions {
 		tipBlockVersionCount := blockVersion.RollingWindowLookBacks[len(blockVersion.RollingWindowLookBacks)-1]
-		if tipBlockVersionCount >= int(activeNetParams.BlockRejectNumRequired) {
+		if templateInformation.BlockVersionCurrent != i && tipBlockVersionCount > mostPopularBlockVersionCount {
 			// Show Green
-			templateInformation.BlockVersionCurrent = i
 			templateInformation.BlockVersionMostPopular = i
 			templateInformation.BlockVersionMostPopularPercentage = toFixed(float64(tipBlockVersionCount)/float64(templateInformation.BlockVersionWindowLength)*100, 2)
-			templateInformation.BlockVersionSuccess = true
-			mostPopularBlockVersionCount = tipBlockVersionCount
-		}
-		if tipBlockVersionCount > mostPopularBlockVersionCount {
-			// Show Red
+			if tipBlockVersionCount >= int(activeNetParams.BlockRejectNumRequired) {
+				templateInformation.BlockVersionSuccess = true
+			}
 			mostPopularBlockVersionCount = tipBlockVersionCount
-			templateInformation.BlockVersionMostPopular = i
-			templateInformation.BlockVersionMostPopularPercentage = toFixed(float64(tipBlockVersionCount)/float64(templateInformation.BlockVersionWindowLength)*100, 2)
 		}
 	}
 	if templateInformation.BlockVersionCurrent == int32(maxVersion) {
```

Comment:
Ah definitely both good options.  Going to merge this as to not block karambles work.  But will open an issue to get back to this first thing Monday morning.